filters/filter_15_DnsFilter/metadata.json

```diff
@@ -20,7 +20,9 @@
     "reference:118",
     "reference:102",
     "reference:13",
-    "reference:16"
+    "reference:16",
+    "obsolete"
   ],
-  "trustLevel": "full"
+  "trustLevel": "full",
+  "disabled": true
```

Comment:
Not sure we should make it `disabled` though, let it continue updating for some time.

Reply:
So, we can add it later?
I left only the tag "obsolete" as you suggested earlier.